examples/global_beacon_idexing.ipynb

```diff
@@ -0,0 +1,321 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Global Beacon"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Global Beacon lets anyone in your organization find datasets based on the entities it contains (i.e. variants, genets, targets). \n",
+    "\n",
+    "`Note`: Only datasets that contain entities can be indexed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Importing SolveBio library and logging in"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Importing SolveBio library\n",
+    "from solvebio import login\n",
+    "from solvebio import Object"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Logging to SolveBio\n",
+    "login(api_host=\"solvebio.api-stag.solvebio.com\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Enabling Global Beacon on dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First let’s start with enabling Global Beacon on the dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "|                  Fields | Data                                               |\n",
+       "|-------------------------+----------------------------------------------------|\n",
+       "|              account_id | 51                                                 |\n",
+       "|     ancestor_object_ids | [u'1661598621921253160']                           |\n",
+       "|            availability | available                                          |\n",
+       "|              class_name | Object                                             |\n",
+       "|              created_at | 2021-11-29T11:24:42.093Z                           |\n",
+       "|     dataset_description | test                                               |\n",
+       "| dataset_documents_count | 26312                                              |\n",
+       "|       dataset_full_name | 1658666726768179211                                |\n",
+       "|              dataset_id | 1658666726768179211                                |\n",
+       "|                   depth | 1                                                  |\n",
+       "|             description | test                                               |\n",
+       "|         documents_count | 26312                                              |\n",
+       "|                filename | BEACONS-TEST-Genes-94-human-mouse-wOXjFS-1426764376|\n",
```

Comment:
It might be nice to rename the dataset to something like `beacon-test-dataset`

Reply:
also put it into a shared vault, I believe you cannot enable beacons on private vault datasets

Reply:
I guess when you say shared vault you mean the `public` vault?

It seems that I don't have proper permissions on the `stag` environment to move the dataset from personal to public vault:
![image](https://user-images.githubusercontent.com/30963594/149304341-6fa76c97-7e7f-46cb-add7-75eede3e328c.png)

Reply:
@nikolamaric gave me permissions, works now.